cluster/addons/pod-resizer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04
```

Comment:
Why ubuntu instead of busybox (like https://github.com/kubernetes/contrib/blob/master/flannel-server/Dockerfile and https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/Dockerfile) or alpine (like https://github.com/kubernetes/contrib/blob/master/ingress/images/haproxy/Dockerfile)?